flow/Makefile

```diff
@@ -199,7 +200,7 @@ SYNTH_SCRIPT ?= scripts/synth.tcl
 
 $(RESULTS_DIR)/1_1_yosys.v: $(DONT_USE_LIBS) $(WRAPPED_LIBS)
 	mkdir -p $(RESULTS_DIR) $(LOG_DIR) $(REPORTS_DIR)
-	($(TIME_CMD) yosys -c $(SYNTH_SCRIPT)) 2>&1 | tee $(LOG_DIR)/1_1_yosys.log
+	($(TIME_CMD) yosys -m $(LSORACLE_PLUGIN) -c $(SYNTH_SCRIPT)) 2>&1 | tee $(LOG_DIR)/1_1_yosys.log
```

Comment:
I think we need an if { [info exists ::env(USE_LSORACLE)] } {} here so that we only run the lsoracle plug in when the switch is set.

Reply:
Good catch, i will update.